### ISSUE TITLE
[Accessibility] Unique title id problem

### ISSUE THREAD

**Issue by lamhuynhorient** [jannicz/ng-svg-icon-sprite#88]
If we add same icon on many places , we get problem about unique id of title .  For below example, we get many title tags with same id 'check-title'
<div >  
  <svg-icon-sprite
   
        [src]="'assets/images/common_sprite.svg#check'"
        [classes]="'icon-30'"
        [title]="'check icon'"
        [attribute]="[
          ['aria-labelledby', 'check-title'],
          ['role', 'img']
        ]"
      ></svg-icon-sprite>
</div>
<div>  
  <svg-icon-sprite
   
        [src]="'assets/images/common_sprite.svg#check'"
        [classes]="'icon-30'"
        [title]="'check icon'"
        [attribute]="[
          ['aria-labelledby', 'check-title'],
          ['role', 'img']
        ]"
      ></svg-icon-sprite>
</div>

**Comment by jannicz** [comment]
Hi. Thanks for using this library. Your problem seems to me to be related with the tag that carries the id="check-title". Remember that in HTML you can only give the an id to one element. Please check via the inspoector whether your markup contains the id check-title" mutlipe times inthe DOM.
I assume that the reference that you create via 'aria-labelledby' cannot be identified by the browser. It does not sound like an error in the svg library here

**Comment by lamhuynhorient** [comment]
@jannicz  If I remove ['aria-labelledby', 'check-title'], the problem still happens. I find the the library auto generate id of title element follow format: 'name of svg icon ' +  '-title' (library get 'name of svg' from src input and '-title' is suffix ). The problem is here, if we have same icon on many places, the library will generate many title element with same id.

For ex: 
 <svg-icon-sprite **[src]="'assets/images/common_sprite.svg#check'"**
    [classes]="'icon-30'"
    [title]="'check icon'"
    [attribute]="[
          ['role', 'img']
    ]"
  ></svg-icon-sprite>
Render :
```
<svg-icon-sprite>
<svg _class="icon-30 ng-star-inserted" width="100%" height="100%" role="img"><title  id="check-title" >correct</title>
<use  xlink:href="assets/images/common_sprite.svg#check"></use></svg>
</svg-icon-sprite>
```

**Comment by jannicz** [comment]
Closing this issue as i don't think it is a problem with the lib but with the approach you took.
